app/javascript/components/notifications/icon.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useReducer, useEffect } from 'react'
+import consumer from '../../application/action_cable_consumer'
+
+function reducer(state, action) {
+  switch (action.type) {
+    case 'notifications.changed':
+      return { count: action.payload.count }
+  }
+}
+
+export function NotificationIcon({ count }) {
+  const [state, dispatch] = useReducer(reducer, { count: count })
+
+  function isUnread() {
```

Comment:
There is no point for this function to exist, at the moment. Do you have different ideas with it?

```jsx
const isUnread = state.count > 0
```

Reply:
I don't actually. Your solution is better.